app/javascript/components/server-components/SubscriptionManager.tsx

```diff
@@ -194,7 +193,7 @@ const SubscriptionManager = ({
     price: Math.round(amountDueToday / product.exchange_rate),
     payInInstallments: subscription.is_installment_plan,
     requireShipping: product.require_shipping,
-    customFields: product.custom_fields,
+    customFields: [],
```

Comment:
I'm a little uncomfortable with this approach - it works now, but this state and form have a ton of logic that may at any time start to rely on custom fields being correct. How about we add a `showCustomFields={false}` prop to `PaymentForm` to make it more explicit?